ignite/distributed/comp_models/native.py

```diff
@@ -371,8 +371,15 @@ def spawn(  # type: ignore[override]
 
             start_processes = mp.spawn
             # start_method and start_processes in pytorch >= 1.5
+            default_start_method = "spawn"
             if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
-                spawn_kwargs["start_method"] = kwargs.get("start_method", "spawn")
+
+                from IPython import get_ipython
+                shell = get_ipython().__class__.__name__
+                if shell == 'ZMQInteractiveShell':
+                    default_start_method = "fork"
```

Comment:
@Chandan-h-509 unfortunately this wont work for those who does not have jupyter installed. Let's try to use
```
in_jupyter = '__IPYTHON__' in globals()
```